openshift/4.0/templates/base-images/README.md

```diff
@@ -0,0 +1,71 @@
+# Base Images Setup
```

Comment:
Unfortunately, these openshift gitops files are out-of-date and being kept for reference only. If you'd like to add some gitops yaml for our image streams, please do that in:
https://github.com/bcgov-c/tenant-gitops-3cd915